bin/set_allowed_mins.py

```diff
@@ -64,7 +64,7 @@ def send_data(db, allowed_minutes, override):
     db.execute(text(query), gradeable=GRADEABLE)
     if override is not None:
         for user in override:
-            query = """INSERT INTO gradeable_allowed_minutes_override (g_id, user_id, allowed_minutes)
+            query = """INSERT INTO gradeable_allowed_minutes_override (g_id, user_id, allowed_minutes)  # noqa: E501
```

Comment:
can you put a comment within a multi-line string?
this seems hazardous

Reply:
Yep, good catch.  I changed the comment to use single quotes and then reverted it without thinking.  I believe that I have fixed the issue.